src/app/psat/pump-fluid/pump-fluid.component.html

```diff
@@ -5,6 +5,12 @@ <h3>Pump</h3>
     </div>
   </div>
 
+  <div class="mx-3 my-0">
```

Comment:
I think the *ngIf here and below may want to be on the div instead of the component. Otherwise there will be two empty divs.